modules/custom/module2/module2.info.yml

```diff
@@ -0,0 +1,5 @@
+name: module2
+description: This is a custom module for module2.
+package: custom
+type: module
+core_version_requirement: ^9.2 || ^10
```

Comment:
Why  ^9.2?

Reply:
It was in the documentation from where I copied this section.